test/helpers/kubectl.go

```diff
@@ -1121,6 +1121,18 @@ func (kub *Kubectl) GetServiceHostPort(namespace string, service string) (string
 	return data.Spec.ClusterIP, int(data.Spec.Ports[0].Port), nil
 }
 
+// GetServiceClusterIPs returns the list of cluster IPs associated with the service. The support
+// for this is only present in later version of Kubernetes(>= 1.20).
+func (kub *Kubectl) GetServiceClusterIPs(namespace string, service string) ([]string, error) {
+	var data v1.Service
+	err := kub.Get(namespace, fmt.Sprintf("service %s", service)).Unmarshal(&data)
```

Comment:
We can replace the `Sprintf` with a pretty simple string concatenation:

```
"service " + service
```

(inspired by https://github.com/cilium/cilium/pull/15089)

Reply:
Good point, will do this. :+1: